pipelines/batch/src/test/java/com/google/fhir/analytics/ReadJsonFilesTest.java

```diff
@@ -103,4 +119,23 @@ public void testProcessBundleRelativeRef()
     assertThat(obs.getSubject().getReference(), equalTo("Patient/749605"));
     assertThat(obs.getEncounter().getReference(), equalTo("Encounter/750983"));
   }
+
+  @Test
+  public void testReadingFromNDJsonFile()
```

Comment:
Thanks for adding this unit-test. Can you please also do an end-to-end test by converting ndjson files to parquet using the pipeline and then update the `TESTED` field? Asking because the current e2e does not cover json input.

Reply:
Ok, did an end-to-end test by converting ndjson files to parquet. Updated the TESTED field with details.